### DISCUSSION TITLE
Assistance with playing video on Annotation clicked.

### DISCUSSION THREAD

**Discussion by skapezempire** [google/model-viewer#4703] [Q&A] [answered]
Hello world, I would need assistance with how to display a video UI when annotations or slots are clicked.

Kindly find the attached image for reference.
Thank you.

![Screenshot 2024-03-06 163428.png](https://github.com/google/model-viewer/assets/18077781/12b2b2b1-02ac-4bcd-91b6-1bc323c9e220)

**Comment by danilokrulj** [comment, marked as answer]
Make a div element with the video inside it and than toggle its display settings when clicking the hotspot?

1) Make the function that toggles the div elements display settings (from "none" to "block" or whatever you use)
2) Find the hotspot by its id.
3) Add a 'click' event listener on it and call the function on click

**Reply by skapezempire** [reply]
Thanks a lot.
